package.json

```diff
@@ -99,43 +138,11 @@
     "prettier": "^2.2.1",
     "prisma": "2.19.0",
     "react-test-renderer": "^17.0.1",
+    "remark-gfm": "^1.0.0",
+    "remark-slug": "^6.0.0",
+    "remark-toc": "^7.2.0",
```

Comment:
`remark-gfm` adds table support.
`remark-slug` adds anchors to titles so you can create links inside document.
`remark-toc` automatically generates table of contents.